Resources/public/css/theme/views/subitemlist.css

```diff
@@ -92,3 +92,69 @@
     left: 50%;
     position: absolute;
 }
+
+.ez-view-subitemlistview .ez-subitemlist-table .ez-subitem-hovered-priority-cell  .ez-subitem-priority-icons:after {
+    content: "\e606";
+    background: #E6E6E6;
+    color: #737373;
+    font-weight: bold;
```

Comment:
it seems this one is always valid so you should get something:
```css
.ez-view-subitemlistview .ez-subitem-priority-icons:after {
    font-weight: bold;
}
```

to not repeat that.